ext/typeexpr/defaults_test.go

```diff
@@ -317,7 +320,11 @@ func TestDefaults_Apply(t *testing.T) {
 				}),
 				cty.ObjectVal(map[string]cty.Value{
 					// No default value for "c" specified, so none applied. The
-					// convert stage will fill in a null.
+					// convert stage will have filled in a null.
```

Comment:
A little more distinct from the previous logic:

```suggestion
					// apply process converts to the target type, filling in a null.
```